orienteer-core/src/main/java/org/orienteer/core/util/StartupPropertiesLoader.java

```diff
@@ -121,4 +122,9 @@ private static Properties retrieveSystemProperties(Properties loadedProperties)
 		loadedProperties.putAll(System.getProperties());
 		return loadedProperties;
 	}
+
+	public static String getAppHome() {
+		String appHome = System.getenv("ORIENTEER_APP_HOME");
```

Comment:
Why, btw `ORIENTEER_APP_HOME`, but not just `ORIENTEER_HOME`?

Reply:
Renamed